project/AppDependencies.scala

```diff
@@ -16,7 +16,7 @@ object AppDependencies {
     "ai.x"                    %% "play-json-extensions"              % "0.42.0",
     "org.typelevel"           %% "cats-core"                         % "2.6.1",
     "com.github.kxbmap"       %% "configs"                           % "0.6.1",
-    "uk.gov.hmrc.objectstore" %% s"object-store-client$playVersion"  % "0.37.0",
+    "uk.gov.hmrc.objectstore" %% s"object-store-client$playVersion"  % "0.39.0",
```

Comment:
this isn't about this dependency, but we could probably update scalmock from 5.1.0 to 5.2.0 below